http3/response_writer.go

```diff
@@ -15,7 +15,12 @@ import (
 	"github.com/quic-go/qpack"
 )
 
-// headerWriter wrap the stream, so first Write will flush header to the stream
+// The maximum length of an encoded quic frame header is 10.
+// 1 (header type) + variable (frame length, the length of the largest encoded quic varint is 9).
```

Comment:
It's 8.

Reply:
The type is not guaranteed to be 1 byte though, is it?

Reply:
16 comes from the original code for encoding frame header. The type is 1 byte for header and data frame according to the code.

It's 8 byte. My mistake.